portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreation.testcase

```diff
@@ -453,13 +469,13 @@ definition {
 
 		task ("View the segment criteria card") {
 			ACSegments.viewSegmentCriteriaIndividualAttributes(
-				attributeName = "jobTitle",
-				attributeValue = '''"Doctor"''',
+				attributeName = "email",
+				attributeValue = '''"testing@liferay.com"''',
 				condition = "does not contain",
 				conjunction = "AND");
 
 			ACSegments.viewSegmentCriteriaIndividualAttributes(
-				attributeName = "doNotCall",
+				attributeName = "User Custom Field",
 				attributeValue = "FALSE",
 				condition = "is",
 				index = "2");
```

Comment:
Hey @manuferreira, I think at the end of the test you could put it to verify that user1 and user2 are not in the list, that way we can verify that the criteria are working in the segment:

`for (var username : list "user1 user1,user2 user2") {
         ACUtils.viewNameListIsNotPresent(nameList = "${username}");
}`

Reply:
I think inside the ACUtils.viewNameListIsNotPresent we already have a for loop.

Reply:
Ohh yes, thanks Manu